lib/bolt_server/transport_app.rb

```diff
@@ -630,6 +634,28 @@ def make_winrm_target(target_hash)
       [400, e.message]
     end
 
+    # Returns a list of targets parsed from a Project inventory
+    #
+    # @param project_ref [String] the project_ref to compute the inventory from
+    get '/project_inventory_targets' do
+      bolt_config = config_from_project(params['project_ref'])
+      if bolt_config.inventoryfile && bolt_config.project.inventory_file.to_s != bolt_config.inventoryfile
+        raise Bolt::ValidationError, "Project inventory must be defined in #{bolt_config.project.inventory_file}"
+      end
+      plugin = Bolt::Plugin.setup(bolt_config, nil)
```

Comment:
Agreed. I don't think that needs to be in this PR though because we're the ones that will be using this endpoint, and we won't be using inventory files with plugins. We'll also be extending this with proper plugin support eventually before a release and part of that work does include handling the built-in plugins.

Reply:
Nit: `plugin` => `plugins` to keep it consistent with https://github.com/puppetlabs/bolt/blob/main/lib/bolt/cli.rb#L332